<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>39</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Showcard Gothic</family>
        <pointsize>22</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>MENU</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>130</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejercicio1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>220</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejercicio 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>310</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejercicio 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejercicio 4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>490</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Finalizar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese un titulo a su tarea:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputTitulo">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="inputDescripcion">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>351</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="inputFecha">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>60</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="date">
       <date>
        <year>2024</year>
        <month>8</month>
        <day>9</day>
       </date>
      </property>
     </widget>
     <widget class="QComboBox" name="tipoTarea">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Diaria</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Semanal</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnAgregarTarea">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese un descripcion a su tarea:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listaTareas">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>300</y>
        <width>601</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Descripcion de la tarea:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>500</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>510</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="seleccionFigura">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Circulo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rectangulo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triangulo</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Selecciona el tipo de figura:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="inputRadio">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el radio para el circulo:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="inputAlto">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="inputAncho">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el alto del rectangulo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el ancho del rectangulo:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="inputPunto1">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>40</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="inputPunto2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>100</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="inputPunto3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>160</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el vertice 1 del triangulo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>80</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el vertice 2 del triangulo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>140</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el vertice 3 del triangulo:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAgregarFigura">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>310</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Figura</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="lienzo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>561</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>90</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ejer3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>510</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>130</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ejer4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>510</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Regresar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
